Site/SiteApplication.php

```diff
@@ -104,15 +104,6 @@ abstract class SiteApplication extends SiteObject
 	 */
 	protected $locale;
 
-	/**
-	 * The Sentry client
-	 *
-	 * @var Raven_Client
-	 *
-	 * @see SiteApplication::getSentryClient()
-	 */
-	protected $sentry_client;
```

Comment:
The upgrading docs say "it's no longer recommended to just use a Client unless you really know what you are doing". https://github.com/getsentry/sentry-php/blob/master/UPGRADE-2.0.md

So I've gone ahead and removed the client and used their global handler as recommended